Airports/EGPF/SMR/Geo.txt

```diff
@@ -4,7 +4,6 @@ N055.52.42.599 W004.25.07.090 N055.52.42.631 W004.25.06.005 smrYellow ;A1
 N055.52.39.081 W004.25.09.844 N055.52.38.628 W004.25.09.177 smrYellow ;A2
 N055.52.16.535 W004.25.40.292 N055.52.16.304 W004.25.39.186 smrYellow ;A3
 N055.52.36.299 W004.25.19.012 N055.52.36.625 W004.25.17.074 smrYellow ;B1
-N055.52.18.606 W004.25.48.600 N055.52.20.279 W004.25.48.678 smrYellow ;C1
```

Comment:
Better just to comment this out with a ; at the start of the line, rather than deleting it